corehq/apps/api/resources/v0_5.py

```diff
@@ -277,15 +278,33 @@ def obj_create(self, bundle, **kwargs):
             validate_profile_required(bundle.data.get('user_profile'), kwargs['domain'])
         except ValidationError as e:
             raise BadRequest(e.message)
+        require_account_confirmation = True if bundle.data.get('require_account_confirmation') == "True" else False
```

Comment:
The typical pattern for clarity and self documentation is to define the fields 
`require_account_confirmation = fields.BooleanField(default=False)`
`send_confirmation_email = fields.BooleanField(default=False)`

And after doing that, bundle.data.get('require_account_confirmation') will already be a boolean value

and then removing it in `def dehydrate()`

Reply:
properly added them as fields on CommCareUserResource, though I'm not sure what you mean by removing them in dehydrate? did you just mean removing them from the returned response?

Reply:
ya, i mean removing them from the returned the response (via dehydrate)

Reply:
removed via dehyrdation [4548870](https://github.com/dimagi/commcare-hq/pull/36675/commits/45488700ab064aaafaed5e6e2c591ad347bf8814)